packaging/google-guest-agent.spec

```diff
@@ -29,58 +29,134 @@ BuildArch: %{_arch}
 BuildRequires: systemd
 %endif
 
+Obsoletes: python-google-compute-engine, python3-google-compute-engine
```

Comment:
Hmm... We may want to consider doing something else here. We may still have a python package (as a library) around.

Reply:
understand this concern, but if we don't remove that package the legacy daemons won't be stopped and deregistered. i think we should update python-gce to stop producing the entrypoint scripts and upload that to pypi for anyone who needs it.

Reply:
Yes. I think that makes sense.